app/views/admin/legal_aid_applications/index.html.erb

```diff
@@ -4,21 +4,44 @@
       back_link: :none
     ) do %>
 
-  <p><%= link_to t('.edit_settings'), admin_settings_path, class: 'govuk-link' %></p>
+  <div id="confirm-delete" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary" data-cancel-text="<%= t('.cancel_delete') %>" hidden>
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      <span id="delete-message"><% t('.warning.delete') %></span>
+    </h2>
```

Comment:
I think you have too many ids in the page. Are they really all necessary? Remember, with with a lot of ids there is a lot of uniqueness to be maintained.